src/core/xds/xds_client/xds_client.cc

```diff
@@ -512,9 +512,8 @@ bool XdsClient::XdsChannel::MaybeFallbackLocked(
     }
     GRPC_TRACE_LOG(xds_client, INFO)
         << "[xds_client " << xds_client_.get() << "] authority " << authority
-        << ": added fallback server " << xds_servers[i]->target()->server_uri()
-        << " (" << authority_state.xds_channels.back()->status().ToString()
-        << ")";
+        << ": added fallback server " << server_uri() << " ("
```

Comment:
This one needs to be `xds_servers[i]->target()->server_uri()`, because it's not going to be the same server used by `this`.